src/composables/multisig.ts

```diff
@@ -0,0 +1,218 @@
+import { computed, ref } from '@vue/composition-api';
+import type { Store } from 'vuex';
+import multisigContract from 'ga-multisig-contract/contracts/SimpleGAMultiSig.aes';
+import { uniqBy } from 'lodash-es';
+import camelcaseKeysDeep from 'camelcase-keys-deep';
+import {
+  MemoryAccount, Universal, Node, Crypto,
+} from '@aeternity/aepp-sdk';
+import { decode } from '@aeternity/aepp-sdk/es/tx/builder/helpers';
+import { IAccount, INetwork, IMultisig } from '../types';
+import {
+  fetchJson,
+  postJson,
+  LOCAL_STORAGE_PREFIX,
+} from '../popup/utils';
+import { useSdk } from './sdk';
+import { createPollingBasedOnMountedComponents } from './composablesHelpers';
+
+interface UseMultisigOptions {
+  /**
+   * TODO: Temporary solution to avoid dependency circle
+   */
+  store: Store<any>
+}
+
+const POLLING_INTERVAL = 5000;
+const MULTISIG_TRANSCATION_EXPARATION_HEIGHT = 50;
+
+const LOCAL_STORAGE_MULTISIG_KEY = `${LOCAL_STORAGE_PREFIX}_multisig`;
+
+function storeMultisig(multisig: IMultisig[]) {
```

Comment:
This method's name stores the GA accounts (multisig vaults) so I suggest to rename it to something more meaningfull.